quickwit/quickwit-cluster/src/cluster.rs

```diff
@@ -626,7 +626,7 @@ mod tests {
             .unwrap(),
         );
         let mut random_generator = rand::thread_rng();
-        let indexing_tasks = (0..10_000)
+        let indexing_tasks = (0..1_000)
```

Comment:
Modifications here are only made to make the test pass as we are now testing the right condition in `wait_untIl_predicate`